src/content/blog/2025-10-29-web-monetization-open-payments-part-1-connecting-wallet.md

```diff
@@ -0,0 +1,175 @@
+---
+title: "How Web Monetization uses Open Payments - Part 1: Connecting Wallet"
+description: "Understanding how the browser extension uses Open Payments to securely connect your wallet for Web Monetization"
+date: 2025-10-29
+slug: web-monetization-open-payments-part-1-connecting-wallet
+authors:
+  - Sid Vishnoi
+author_urls:
+  - https://sidvishnoi.com?ref=ilf_engg_blog
+tags:
+  - Open Payments
+  - Web Monetization
+---
+
+Welcome to the first of a three-part series on how Open Payments powers Web Monetization. We’ll also explore the mechanics of the Web Monetization browser extension, which serves as a temporary agent until web browsers natively support this technology.
+
+I assume a conceptual understanding of the underlying technologies — the [Web Monetization](https://webmonetization.org) standard and the supporting [Open Payments API](https://openpayments.dev) — which will be technically connected for you throughout the series.
+
+This article focuses on the initial extension and wallet setup. I treat you, the reader, as the paying party here — supporting the content you love.
+
+The next two articles will then shift to explain the actual money-sending mechanics and related topics. They will be from the perspective of the Web Monetization Agent (the browser extension here), as it automatically handles the payment processes on your behalf.
+
+## Parts of the Extension
+
+A browser extension consists of several key components. The pop-up is the user interface that appears when users click the extension icon in the browser's toolbar. The pop-up is typically the main interface that users interact with. In the Web Monetization extension, you will first use this pop-up to enter your wallet address and set a budget for the wallet connection process. Later, you can check a website's monetization status and send one-time payments there.
+
+The background script (also known as the background service worker) functions as a secure backend server within your browser. It coordinates activities and securely stores sensitive data, such as payment information, away from the web pages. The Open Payments bits of the extension are live here.
```

Comment:
Just wanted to call out the use of 'live' at the end of the sentence. It's a heteronym, so I wasn't sure if you meant "...bits of the extension live here" or as it's written. Both are fine :)